src/ast/passes/semantic_analyser.cpp

```diff
@@ -1178,6 +1227,37 @@ void SemanticAnalyser::visit(Call &call)
 
     call.type = CreateUInt64();
   }
+  else if (call.func == "offsetof")
+  {
+    if (check_nargs(call, 2))
+    {
+      auto element = dynamic_cast<Identifier *>(call.vargs->at(1));
+      if (element == nullptr)
+      {
+        LOG(ERROR, call.loc, err_)
+            << call.func << "() only support single word right now.";
+      }
```

Comment:
This branch is an internal error, no? We do error checking and coercion above. Maybe it should just panic or throw an internal exception.

Reply:
I just remove it in newest force push.